faq/faq.md

```diff
@@ -0,0 +1,351 @@
+employer) instantly gain copyright over it.
+
+Creative Commons provides a better and lengthier answer in their [Frequently
+Asked
+Questions](https://creativecommons.org/faq/#what-is-copyright-and-why-does-it-matter).
+
+## What is a license? {#what-is-license}
+
+One problem with copyright as it pertains to software is that it makes software
+unshareable by default. A license changes that. A license defines the terms
+under which the copyright holder allows the recipient of the license to use the
+software.
+
+If the license allows the recipient to [use, study, share and
+improve](https://fsfe.org/freesoftware/basics/4freedoms.html) the software, then
+that software is Free Software.
+
+## Which license should I choose? {#which-license}
+
+## Do I really need to include the license and copyright headers in all files? Why should I care? {#why-care}
+
+## What is SPDX? {#what-is-spdx}
+
+SPDX stands for [Software Package Data Exchange](https://spdx.org/). It is a
+project by the [Linux Foundation](https://www.linuxfoundation.org/) and the rock
+upon which REUSE is built. SPDX defines a standardized way to share copyright
+and licensing information between projects and people. Most importantly for
+REUSE, SPDX maintains the [SPDX License List](https://spdx.org/licenses/), which
+defines standardized identifiers for a lot of licenses.
+
+# The tool {#tool}
+
+## How do I install and use the REUSE tool? {#install-tool}
+
+The REUSE tool is a handy companion that can verify your compliance with REUSE
+and automate a lot of adjacent tasks. You can find the full documentation for
+the tool at <https://reuse.readthedocs.io/>. But the short of it is:
+
+```
+$ pip3 install --user fsfe-reuse 
+$ export PATH=~/.local/bin:$PATH
+$ reuse --help
+```
+
+## How do I exclude a file from REUSE compliance testing? {#exclude-file}
+
+If the file is a build artifact and you use Git, simply make sure that the file
+is covered by your `.gitignore` file.
+
+Otherwise, you cannot exclude files from REUSE compliance testing. It would go
+entirely against the purpose of REUSE: making sure that every file has a
+copyright and license tag.
+
+If you really want to exclude a file, consider using the
+[CC0](https://creativecommons.org/publicdomain/zero/1.0/) license for this
+file.  By doing this, you put the file in the public domain, or your country's
+equivalent.
+
+There is one exception: License files are automatically excluded from compliance
+testing.
+
+<!-- FIXME: Link to uncopyrightable question and DEP5 question -->
+
+## Do you support a version control system other than Git? {#no-git}
+
+Currently we do not, but please [get in
+touch](https://github.com/fsfe/reuse-tool/issues).
+
+# Licensing and compliance {#licensing}
+
+## Which licenses are compatible? {#compatible-licenses}
+
+## Is there a standard format for declaring copyright? {#standard-copyright}
+
+Copyright year copyright holder email
```

Comment:
For adoption reasons I wouldn't mandate the use of "©". People are _really really really_ bad at using keyboards, and don't know how to write an accented character (ex., ŭ, ü, û, ú, ù, ũ), let alone some Unicode symbol.

I understand the rationale, and I loathe that people suck at using keyboards, but it would be really bad for adoption. Moreover, most jurisdictions don't mandate copyright notices _anyway_, so the internationally-agreed-upon © isn't strictly needed in those jurisdictions.

Reply:
I would be happy if we could drop © since it makes things easier, but have everything supported by the linter.

If dropping © does not cause severe problems for projects adopting our spec, I would be comfortable with going this way.

Reply:
I’m OK with not mandating it, but leaving it as one of the few options.